src/main/resources/application.properties

```
@@ -1,4 +1,8 @@
-spring.datasource.url=
-spring.datasource.username=
-spring.datasource.password=
+spring.datasource.url=jdbc:postgresql://localhost:5433/telegram-bot
+spring.datasource.username=postgres
+spring.datasource.password=12345678
+spring.jpa.show-sql=true
+server.port=8081
+spring.jpa.hibernate.ddl-auto=update
```

```suggestion
spring.jpa.hibernate.ddl-auto=validate
```
БД должны инициализировать через liquibase, поэтому тут указываем validate